addons/l10n_ar_withholding/models/account_tax.py

```diff
@@ -7,13 +7,36 @@ class AccountTax(models.Model):
     _inherit = 'account.tax'
 
     l10n_ar_withholding_payment_type = fields.Selection(
-        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding type',
-        compute="_compute_l10n_ar_withholding_payment_type", store=True, readonly=False)
-
+        [('supplier', 'Vendor Payment'), ('customer', 'Customer Payment')], 'AFIP Argentinean Withholding Type',
+        compute="_compute_l10n_ar_withholding_payment_type", store=True, readonly=False, help="Withholding tax for supplier or customer payments.")
+    l10n_ar_tax_type = fields.Selection(string='Argentinean Tax Type', selection=[
+        ('earnings_withholding', 'Earnings'),
```

Comment:
in each of the selections there is _withholding in the type, while all of the selections are withholding